lib/awful/screenshot.lua

```diff
@@ -0,0 +1,884 @@
+---------------------------------------------------------------------------
+--- Screenshots and related configuration settings
+--
+-- @author Brian Sobulefsky &lt;brian.sobulefsky@protonmail.com&gt;
+-- @copyright 2021 Brian Sobulefsky
+-- @inputmodule awful.screenshot
+---------------------------------------------------------------------------
+
+-- Grab environment we need
+local capi = { root = root,
+               screen = screen,
+               client = client,
+               mousegrabber = mousegrabber }
+local aw_screen = require("awful.screen")
+local gears = require("gears")
+local beautiful = require("beautiful")
+local wibox = require("wibox")
+local cairo = require("lgi").cairo
+local naughty = require("naughty")
+
+-- The module to be returned
+local module = { mt = {} }
+-- The screenshow object created by a call to the screenshow module
+local screenshot = {}
+-- The various methods of taking a screenshow (root window, client, etc).
+local screenshot_methods = {}
+
+-- Configuration data
+local ss_dir = nil
+local ss_prfx = nil
+local frame_color = gears.color("#000000")
```

Comment:
module.frame_color = ....

Reply:
I have updated these three tentatively (this is how they function if no argument provides a value)
```
-- Configuration data
local global_screenshot_dir = nil
local global_screenshot_prefix = nil
local global_frame_color = gears.color("#000000")
local initialized = nil
```

Reply:
using module properties would be better, but now it looks at least readable 👍